app/js/containers/ProfileContainer.jsx

```diff
@@ -22,7 +22,7 @@ class ProfileContainer extends Component {
         <SellerInformation reputation={profile.reputation} address={profile.address} name={profile.username}/>
         <Trades trades={profile.trades}/>
         <Offers offers={profile.offers} location={profile.location} />
-        <StatusContactCode value={profile.statusContactCode} />
+        <StatusContactCode value={profile.statusContactCode} toggleQRCode={this.props.toggleQRCode} showQRCode={this.props.showQRCode} />
```

Comment:
why do you use redux to do this action?
I might be wrong but I feel this can be done locally to the component no?